src/web/lib/hakuneko/frontend@classic-dark/app.html

```diff
@@ -158,8 +174,15 @@
             /**
              *
              */
-            getPagePanelStyle(readerEnabled) {
-                return ( readerEnabled ? '' : 'display: none;' );
+            getPagePanelStyle(readerEnabled,chapter) {
+                return ( !readerEnabled || chapter==undefined ? 'display: none;' : '' );
```

Comment:
Use type related comparison `===`, otherwise `null == undefined` will evaluate to `true`